src/analysis/fallbackAnalysisAdapter.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import {Graph} from "../core/graph";
+import type {RepoId} from "../core/repoId";
+import type {PluginDeclaration} from "./pluginDeclaration";
+import type {IAnalysisAdapter} from "./analysisAdapter";
+
+import {fallbackDeclaration} from "./fallbackDeclaration";
+
+export class AnalysisAdapter implements IAnalysisAdapter {
```

Comment:
The name `FallbackAnalysisAdapter` (maybe dropping the `Analysis`) makes more sense to me. Thoughts?

Reply:
Done.